test/smoke/src/main.ts

```diff
@@ -53,7 +54,7 @@ const opts = minimist(args, {
 
 const workspaceFilePath = path.join(testDataPath, 'smoketest.code-workspace');
 const testRepoUrl = 'https://github.com/Microsoft/vscode-smoketest-express';
-const workspacePath = path.join(testDataPath, 'vscode-smoketest-express');
+const folderPath = path.join(testDataPath, 'vscode-smoketest-express');
```

Comment:
From @joaomoreno 

@sandy081 `folderPath` should be `workspaceFolderPath`.

Reply:
`workspaceFolderPath` is confusing. It looks like `App` has `folderPath` and `workspaceFilePath` and it can run with either of these by using `workspacePath`. It was very confusing before with `workspacePath` and `workspaceFilePath` and `workspacePath` is being always folder